tools/reviewer/service/CodeReviewService.java

```diff
@@ -51,14 +50,20 @@
   @FlagDesc(name = "firestore_review_root", description = "Review root path in Firestore")
   private static final Flag<String> firestoreReviewRoot = Flag.create("/reviewer");
 
+  private static final String META_COLLECTION = "/reviewer-meta";
```

Comment:
Firestore has a `collection/document/collection/document...` structure.
Let's have the following structure:
`reviewer/data/diff/<diff id>/<diff>`
`reviewer/data/diff` is a `document`, and can therefore have fields. `last_diff_id` can be a field in that document.